### PR TITLE
Release sdk java 7.0.0-alpha5.1

### PR DESCRIPTION
**Pull request by fjtirado**
**Many thanks for submitting your Pull Request :heart:!**

**What this PR does / why we need it**:

**Special notes for reviewers**:

**Additional information (if needed):**

### CODE REVIEW

File: content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+date: 2024-11-7
+description: >
+   Heads up, community! We have released a new Java SDK version compatible with Specification 1.0.0.
+---
+
+## Announcing the Release of Serverless Workflow Java SDK 7.0.0-alpha1
+
+Hello, Serverless Workflow community!
+
+We're excited to announce the release of the latest Java SDK, version 7.0.0-alpha5, now compatible with the upcoming Serverless Workflow Specification 1.0.0. You can dive right in by using [one of the examples](https://github.com/serverlessworkflow/specification/tree/main/examples) available in the specification repository. Here's a quick example to get you started:
```
Comment (ricardozanini) [inline content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md:13]:
```suggestion
We're excited to announce the release of the latest Java SDK, version 7.0.0-alpha5.1, now compatible with the upcoming Serverless Workflow Specification 1.0.0. You can dive right in by using [one of the examples](https://github.com/serverlessworkflow/specification/tree/main/examples) available in the specification repository. Here's a quick example to get you started:
```

File: content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+{{< /card >}}
+
+To install the SDK, simply add it as a dependency to your Maven project:
+
+{{< card code=true header="**Maven Dependency**" lang="Java" >}}
+<dependency>
+    <groupId>io.serverlessworkflow</groupId>
+    <artifactId>serverlessworkflow-api</artifactId>
+    <version>7.0.0-alpha5</version>
```
Comment (ricardozanini) [inline content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md:29]:
```suggestion
    <version>7.0.0-alpha5.1</version>
```

File: content/en/blog/releases/release-sdk-java-7.0.0-alpha5.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+title: Serverless Workflow Java SDK 7.0.0-alpha5-1
 author: Francisco Javier Tirado Sarti
-date: 2024-11-7
+date: 2024-12-4
 description: >
    Heads up, community! We have released a new Java SDK version compatible with Specification 1.0.0.
 ---
 
-## Announcing the Release of Serverless Workflow Java SDK 7.0.0-alpha1
+## Announcing the Release of Serverless Workflow Java SDK 7.0.0-alpha5.1
```
Comment (ricardozanini) [inline content/en/blog/releases/release-sdk-java-7.0.0-alpha5.md:9]: I think we shouldn't change this blog. Is it a confusion? hehe

File: content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+title: Serverless Workflow Java SDK 7.0.0-alpha1
+author: Francisco Javier Tirado Sarti
+date: 2024-11-7
+description: >
+   Heads up, community! We have released a new Java SDK version compatible with Specification 1.0.0.
+---
+
+## Announcing the Release of Serverless Workflow Java SDK 7.0.0-alpha1
```
Comment (ricardozanini) [inline content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md:9]:
```suggestion
## Announcing the Release of Serverless Workflow Java SDK 7.0.0-alpha5.1
```

File: content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
+---
+title: Serverless Workflow Java SDK 7.0.0-alpha1
```
Comment (ricardozanini) [inline content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md:2]:
```suggestion
title: Serverless Workflow Java SDK 7.0.0-alpha5.1
```

File: content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+author: Francisco Javier Tirado Sarti
+date: 2024-11-7
```
Comment (ricardozanini) [inline content/en/blog/releases/release-sdk-java-7.0.0-alpha5.1.md:4]:
```suggestion
date: 2024-12-04
```

**Review by ricardozanini — APPROVED** [review]

**Review by cdavernas — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**